docs/usage/ui/3_management.md

```diff
@@ -4,5 +4,45 @@ sidebar_position: 3
 ---
 
 # Subnet Management
```

Comment:
# Subnet Management

Subnet management is used for adding and removing Masternodes in the Subnet. To manage the subnet, you need to use the Grandmaster Account, as only the Grandmaster has the right to manage the Subnet.

You can find the Grandmaster Key in the keys.json file.

After making a modification with Subnet management, the change will take effect in the next epoch (900 blocks).
When adding a Masternode address in the management, the new Masternode server should also be started up and added to the network.